tests/testthat/test_randomPlantedForest_classif_rpf.R

```diff
@@ -0,0 +1,15 @@
+test_that("autotest", {
+  learner = lrn("classif.rpf")
+  expect_learner(learner)
+  # note that you can skip tests using the exclude argument
+  result = run_autotest(learner)
+  # FIXME: Response is different for different predict types
```

Comment:
i this addressed by now? why is the test passing?

Reply:
This appears to be fixed, so I'm removing the comment